web/api/Dockerfile

```diff
@@ -1,3 +1,4 @@
-FROM osm_base
+ARG BASE_IMAGE=osm/osm_base
```

Comment:
I think this is reasonable but just make sure to document how a person building the images locally would need to tag the base image for this to work.